src/phoenix/trace/exporter.py

```diff
@@ -30,6 +32,52 @@ def export(self, _: Any) -> None:
         pass
 
 
+class _OpenInferenceExporter:
+    def __init__(
+        self,
+        endpoint: Optional[str] = None,
+        host: Optional[str] = None,
+        port: Optional[int] = None,
```

Comment:
I think that we ought to stick to fewer abstractions, even if that makes the code not DRY, especially when there isn't  an existing issue that abstraction is addressing specifically.